### DISCUSSION TITLE
How to change the default gtd for PE executables

### DISCUSSION THREAD

**Discussion by Wall-AF** [NationalSecurityAgency/ghidra#6613] [Q&A]
I'm REing code compiled by MSVC 4.2 compiler and I've kinda created an appropriate `.gdt` file.  How do I make Ghidra pick my `.gdt` instead of `windows_vs12_32.gdt` when I open new DLLs to reverse?

Also, if I regenerate my file using the same name, Ghidra thinks it's different and the associations get broken.  What can I do here?

**Comment by ryanmkurtz** [comment]
I think it's in the Analysis Options for the "Apply Data Archives" analyzer.
